nominal/sdk.py

```diff
@@ -0,0 +1,403 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from types import MappingProxyType
+from typing import BinaryIO, Iterable, Literal, Mapping, Sequence, TextIO
+
+from conjure_python_client import RequestsClient, ServiceConfiguration
+
+from ._api.combined import attachments_api
+from ._api.combined import scout
+from ._api.combined import scout_catalog
+from ._api.combined import scout_run_api
+from ._api.ingest import upload_api
+
+IntegralNanosecondsUTC = int
+IntegralNanosecondsRelative = int
+
+
+class AbsoluteTimestamp:
```

Comment:
this might make more sense as a few functions rather than a method that just stores a single int.